### PR TITLE
fix: Git root URI should not be absolute for Windows

### LINKED ISSUE #93: Error on Windows - shell cannot spawn
`A system error occurred (spawn C:\WINDOWS\system32\cmd.exe ENOENT)`

I get this error when trying to generate a message on windows

### PR DESCRIPTION
**Pull request by MichaelCurrin**
Resolves #93

### CODE REVIEW

File: src/git/cli.ts [hunk 1]
```diff
@@ -36,7 +36,10 @@ async function _diffIndex(
   repository: Repository,
   options: string[] = []
 ): Promise<Array<string>> {
-  const cwd = repository.rootUri.path;
+  let cwd = repository.rootUri.path;
+  // Hack to make handle Windows path from rootUri, since it should not appear as a root path.
+  cwd = cwd.replace("/c:/", "c:/");
+
```
Comment (yi-Xu-0100) [inline src/git/cli.ts:42]:
```suggestion
  let cwd = repository.rootUri.fspath;
```

**Review by yi-Xu-0100 — COMMENTED** [review]
It may fix the path issue in windows.

File: src/git/cli.ts (hunk 1 above)
Comment (MichaelCurrin) [reply src/git/cli.ts:42]: Great, thanks!

**Comment by MichaelCurrin** [comment]
oh it's actually `fsPath` not `fspath`

**Pull request merged**